docs/user/tutorials/quickstart.md

```diff
@@ -0,0 +1,138 @@
+# Quickstart
+
+This guide covers how you can quickly get started using Hypper.
+
+If you already know Helm, this will be a breeze, as Hypper follows Helm's UI,
+being a drop-in replacement, and extends its functionalities to install Helm
+and Hypper charts alike.
+
+## Prerequisites
+
+The following prerequisites are required for a successful and properly secured
+use of Hypper.
+
+1. A Kubernetes cluster.
+1. Deciding what security configurations to apply to your installation, if any
+1. Installing and configuring Hypper.
+
+
+## Install Kubernetes or have access to a cluster
```

Comment:
should we link to the official k8s docs here?

Reply:
Either that or rancher docs....

Reply:
There are so many ways to get a Kubernetes cluster. We shouldn't be telling people how to do it. People who come to check this out will already have a way to get Kubernetes.